.github/workflows/ci.yml

```diff
@@ -26,6 +26,8 @@ jobs:
             ${{ runner.os }}-gradle-
       - name: Ktlint
         run: ./gradlew ktlint
+      - name: Detekt
+        run: ./gradlew detekt
```

Comment:
this will for now just run on `stripe-core`. As more core modules start applying the `core-library` base config, they'll get `Detekt` runs automatically.